sql/expression/arithmetic.go

```diff
@@ -45,7 +45,19 @@ func arithmeticWarning(ctx *sql.Context, errCode int, errMsg string) {
 	})
 }
 
-// Arithmetic expressions (+, -, *, ...) DOES NOT INCLUDE "/" as it has its own function
+// ArithmeticOp implements an arithmetic expression. Since we had separate expressions
+// for division and mod operation, we need to group all arithmetic together. Use this
+// expression to define any arithmetic operation that is separately implemented from
+// Arithmetic expression in the future.
+type ArithmeticOp interface {
+	sql.Expression
```

Comment:
If this new interface uses `BinaryExpression` instead of just `sql.Expression`, are we able to simplify and not create the `LeftChild()` and `RightChild()` members?